### PR TITLE
Update connectiq from 3.1.8-2020-05-01-5a72d0ab2 to 3.2.3-2020-10-13-…

### PR DESCRIPTION
**Pull request by phedoreanu**
…c14e609bdu

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.

### CODE REVIEW

**Comment by miccal** [comment]
So the `.app` name has changed from `ConnectIQ.app` to `MonkeyMotion.app`?

**Comment by phedoreanu** [comment]
No, there are two apps inside the SDK. 

Line 10 remains unchanged (app "connectiq-sdk-mac-#{version}/bin/ConnectIQ.app").

File: Casks/connectiq.rb [hunk 1]
```diff
@@ -1,15 +1,28 @@
...
+  artifact "connectiq-sdk-mac-#{version}/samples", target: "#{HOMEBREW_PREFIX}/Cellar/#{cask}/#{version}/samples"
+  artifact "connectiq-sdk-mac-#{version}/resources", target: "#{HOMEBREW_PREFIX}/Cellar/#{cask}/#{version}/resources"
+  artifact "connectiq-sdk-mac-#{version}/ConnectIQBasics.html", target: "#{HOMEBREW_PREFIX}/Cellar/#{cask}/#{version}/ConnectIQBasics.html"
+  artifact "connectiq-sdk-mac-#{version}/CoreTopics.html", target: "#{HOMEBREW_PREFIX}/Cellar/#{cask}/#{version}/CoreTopics.html"
+  artifact "connectiq-sdk-mac-#{version}/FAQ.html", target: "#{HOMEBREW_PREFIX}/Cellar/#{cask}/#{version}/FAQ.html"
+  artifact "connectiq-sdk-mac-#{version}/LearningMonkeyC.html", target: "#{HOMEBREW_PREFIX}/Cellar/#{cask}/#{version}/LearningMonkeyC.html"
+  artifact "connectiq-sdk-mac-#{version}/README.html", target: "#{HOMEBREW_PREFIX}/Cellar/#{cask}/#{version}/README.html"
+  artifact "connectiq-sdk-mac-#{version}/ReferenceGuides.html", target: "#{HOMEBREW_PREFIX}/Cellar/#{cask}/#{version}/ReferenceGuides.html"
+  artifact "connectiq-sdk-mac-#{version}/UserExperienceGuidelines.html", target: "#{HOMEBREW_PREFIX}/Cellar/#{cask}/#{version}/UserExperienceGuidelines.html"
```
Comment (reitermarkus) [inline Casks/connectiq.rb:27]: Casks shouldn't install into `Cellar`.

Comment (phedoreanu) [reply Casks/connectiq.rb]: Replaced `Cellar` with `Caskroom`. Thanks.

Comment (reitermarkus) [reply Casks/connectiq.rb]: These are installed into the `Caskroom` in any case, all of these `artifact`s are unneeded.

Comment (phedoreanu) [reply Casks/connectiq.rb]: Pushed.

**Comment by miccal** [comment]
Thank you for your contribution to `homebrew-cask` @phedoreanu.

**Review by miccal — APPROVED** [review]

**Pull request merged**